.gitignore

```diff
@@ -4,6 +4,8 @@
 *.dll
 *.so
 *.dylib
+*.mod
+*.sum
```

Comment:
https://www.reddit.com/r/golang/comments/97dxj1/should_i_add_the_gosum_file_to_the_repo_of_a/

https://stackoverflow.com/questions/53837919/should-go-sum-file-be-checked-in-to-the-git-repository

Reply:
Thanks, thats really helpful. We had some conversation about this, but did't come to conclusion it's worth checking in the internet. :)

Reply:
No problem - otherwise you'd have no way to ensure you use same version of dependencies 😜